qemu_v8.mk

```diff
@@ -189,6 +256,35 @@ soc-term:
 soc-term-clean:
 	$(MAKE) -C $(SOC_TERM_PATH) clean
 
+################################################################################
+# mkimage - create images to be loaded by U-Boot
+################################################################################
+# Without the objcopy, the uImage will be 10x bigger.
+$(KERNEL_UIMAGE): linux | $(BINARIES_PATH)
```

Comment:
`$(KERNEL_UIMAGE): u-boot linux | $(BINARIES_PATH)`

(`mkimage` is generated by target `u-boot`)